stable/redis/values.yaml

```diff
@@ -0,0 +1,33 @@
+## Bitnami Redis image version
+## ref: https://hub.docker.com/r/bitnami/redis/tags/
+##
+image: bitnami/redis:3.2.4-r0
```

Comment:
Mostly a curiosity thing, but what is different in bitnami/redis vs the official redis image?